### ISSUE TITLE
How to know how many tag in a SVG file

### ISSUE THREAD

**Issue by s-triar** [riclolsen/scadavis#1]
From the example scadavis change the value of the TAG like `sv.storeValue("TAG2", 100 + Math.random() * 80);`.

But, How do I know the SVG file has TAG?

**Comment by riclolsen** [comment]
If you do not know beforehand, you can call sv.getTagList().
https://scadavis.io/synoptic/apidoc/global.html#getTagsList

**Comment by s-triar** [comment]
So I must use the SCADAvis.io Synoptic Editor to give TAG to a SVG file because plain SVG file only has path. is that correct?

**Comment by riclolsen** [comment]
Yes, you have to use the editor to create markup for linking tag values with svg properties that will be animated at runtime.
